front/app/components/admin/Graphs/PointMap/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useMemo, useEffect } from 'react';
+
+import { when as reactiveUtilsWhen } from '@arcgis/core/core/reactiveUtils.js';
+import Point from '@arcgis/core/geometry/Point';
+import Graphic from '@arcgis/core/Graphic';
+import FeatureLayer from '@arcgis/core/layers/FeatureLayer';
+import Renderer from '@arcgis/core/renderers/SimpleRenderer';
+import SimpleMarkerSymbol from '@arcgis/core/symbols/SimpleMarkerSymbol';
+import MapView from '@arcgis/core/views/MapView';
+import { colors } from '@citizenlab/cl2-component-library';
+
+import { IMapConfig } from 'api/map_config/types';
+
+import useLocalize from 'hooks/useLocalize';
+
+import EsriMap from 'components/EsriMap';
+import { applyHeatMapRenderer, parseLayers } from 'components/EsriMap/utils';
+
+interface Props {
+  points: GeoJSON.Point[];
+  layerTitle: string;
+  mapConfig?: IMapConfig;
+  layerId?: string;
+  heatmap?: boolean;
+  onInit?: (mapView: MapView) => void;
+}
+
+const circleSymbol = new SimpleMarkerSymbol({
+  color: colors.primary,
+  style: 'circle',
+  size: '18px',
+  outline: {
+    color: colors.white,
+    width: 2,
+  },
+});
+
+const PointMap = ({
```

Comment:
Created this component so I can share functionality between both `PointLocationQuestion` components (the one you made and the new one in the report builder)